app/scripts/app.ts

```diff
@@ -1,4 +1,5 @@
 ///<reference path="./modules/core/src/types/index.d.ts" />
+/* eslint-disable @spinnaker/import-sort */
```

Comment:
Explicitly turning off to ensure existing import order that relies on side effects.